src/App.tsx

```diff
@@ -67,10 +67,9 @@ const App: FC = () => {
 
   const dispatch = useDispatch();
   const user = useSelector((store: RootStore) => store.user);
-  // Authority/Orcid modal should always be opened on first login
-  const [showAuthorityOrcidModal, setShowAuthorityOrcidModal] = useState(!localStorage.getItem('previouslyLoggedIn'));
   const [isLoadingUser, setIsLoadingUser] = useState(true);
   const [authorities, isLoadingAuthorities, handleNewSearchTerm] = useFetchAuthorities(user.name);
+  const [authorityDataUpdated, setAuthorityDataUpdated] = useState(false);
```

Comment:
er denne egentlig nødvendig? virker som den kun blir satt hvis vi har authorities. Kan vi ikke heller bare fjerne denne, og sjekke på authorities på linje 151?

Reply:
ikke helt, 3 scenario for å få opp modal:
1. brukeren har authority (user.authority)
2. brukeren har potensielle authorities (user.possibleAuthorities)
3. brukeren må opprette ny authority

vi kan fange 1 og 2 med `user.authority || user.possibleAuthorities.length > 0`, men vi får ikke fanget 3 siden vi av en eller annen grunn har emptyUser i redux _før_ man har logget inn i stedet for null :/
kan lage en ny task på å fjerne emptyUser, og nevne dette der også

Reply:
ah, ok. ja, sikkert best å sette null isteden for emptyUser ja